src/index.html

```diff
@@ -1,17 +1,217 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="forms">
+        <legend>Personal information</legend>
+        <div class="formField">
+          <label for="surrname">
+            Surname:
+          </label>
+          <input
+            type="text"
+            name="surrname"
+            id="surrname"
+            required
+          >
+        </div>
+
+        <div class="formField">
+          <label for="name">
+            Name:
+          </label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            required
+          >
+        </div>
+
+        <div class="formField">
+          <label for="age">
+            How old are You?
+          </label>
+          <input
+          type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="12"
+            max="100"
+            required
+          >
+        </div>
+
+        <div class="formField">
+          <label for="birth">
+            Full date of birth:
+          </label>
+          <input
+            type="date"
+            name="birth"
+            id="birth"
+            required
+          >
+        </div>
+
+        <div class="formField">
+          <label for="termsOfAgreement">
+            I accept the term of the agreement
+          </label>
+          <input
+            type="checkbox"
+            name="termsOfAgreement"
+            id="termsOfAgreement"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="forms">
+        <legend>Registration</legend>
+        <div class="formField">
+          <label for="email">
+            E-mail:
+          </label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="formField">
+          <label for="password">
+            Password:
+          </label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="8"
+            maxlength="16"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="forms">
+        <legend>An interesting fact about you!</legend>
+        <div class="formField">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cats-lover"
+            id="cats-lover1"
```

Comment:
Make sure to be consistent with your id's case, if you're using camelCase for id's, use it everywhere, and in same for other cases. Also 1 suffix is not describing enough, use something like yes or no
```suggestion
            id="catsLoverYes"
```